src/Parser/HolyPriest/Modules/PriestCore/Serendipity.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage, formatNumber } from 'common/format';
+
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+
+// dependencies
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SerenityReduction from './SerendipityReduction/SerenityReduction';
+import SanctifyReduction from './SerendipityReduction/SanctifyReduction';
+
+// This module might contain too much information but to rewrite some of the output
+// in this file would entail a lot of redundant copying (aka violating the "DRY"
+// coding mindset). Let me @enragednuke if you think I should move stuff out.
+
+// Currently contains:
+// - "Wasted Serendipity" statistic
+// - "Fully missed holy word casts (via Serendipity)" suggestion
+// - "Holy Priest T20 2P Bonus" item
+class Serendipity extends Module {
+  static dependencies = {
+    combatants: Combatants,
+    serenity: SerenityReduction,
+    sanctify: SanctifyReduction,
+  }
+
+  statistic() {
+    const formatted = {
+      'serenity': {
+        'wasted': formatNumber(this.serenity.overcast / 1000),
+        'raw': formatNumber(this.serenity.rawReduction / 1000),
+      },
+      'sanctify': {
+        'wasted': formatNumber(this.sanctify.overcast / 1000),
+        'raw': formatNumber(this.sanctify.rawReduction / 1000),
+      },
+      'asCasts': {
+        'serenity': (this.serenity.overcast / 1000) / this.serenity.maxCooldown,
+        'sanctify': (this.sanctify.overcast / 1000) / this.sanctify.maxCooldown,
+      },
+    };
+
+    const percWastedVersusTotal = (this.serenity.overcast + this.sanctify.overcast) / (this.serenity.rawReduction + this.sanctify.rawReduction);
+    console.log(percWastedVersusTotal);
```

Comment:
put log lines behind a const 'debug' boolean, or remove entirely

Reply:
forgot about it